libs/MailDB/include/MailDB/PgMailDB.h

```diff
@@ -17,32 +17,20 @@ class PgMailDB : public IMailDB
 
     virtual ~PgMailDB();
 
-    // TODO: Viacheslav
     virtual void Connect(const std::string &connection_string) override;
     virtual void Disconnect() override;
     virtual bool IsConnected() const override;
 
-    // TODO: Denys
-    virtual void SignUp(const std::string_view user_name, const std::string_view hash_password) override;
     virtual void Login(const std::string_view user_name, const std::string_view hash_password) override;
-    virtual std::string GetPasswordHash(const std::string_view user_name) override;
 
 
-    // TODO: Viacheslav
     virtual std::vector<User> RetrieveUserInfo(const std::string_view user_name = "") override;
     virtual std::vector<std::string> RetrieveEmailContentInfo(const std::string_view content = "") override;
     virtual void InsertEmail(const std::string_view sender, const std::string_view receiver,
                                 const std::string_view subject, const std::string_view body) override;
     virtual void InsertEmail(const std::string_view sender, const std::vector<std::string_view> receivers,
                                 const std::string_view subject, const std::string_view body) override;
 
-    // TODO: Denys
-    virtual std::vector<Mail> RetrieveEmails(const std::string_view user_name, bool should_retrieve_all = false) const override;
```

Comment:
std::string additional_condition = ""; here is a redudant string initialization
range-base loop variables is copied but only used as const reference, consider making it a const reference